src/documents/FrameworkConcepts.md

```diff
@@ -8,31 +8,56 @@ The Fluid Infusion Framework is built on a number of important concepts. This pa
 
 ## Model Objects
 
-The greatest goal of the framework is *transparency*—data is pure data, and markup is pure markup. Infusion doesn't impose any inheritance hierarchies or strict contracts on how you structure your data model.
+The greatest goal of the framework is *transparency* &#8212; data is pure data, and markup is pure markup. Infusion doesn't impose any inheritance hierarchies or strict contracts on how you structure your data model.
 
-In Infusion, a tree of objects has the status of being a Model Object if it can successfully be copied by the framework utility function `fluid.copy()`. This function duplicates an object tree in a very clear way: each object is cloned recursively, except for immutable values such as strings, numbers, functions, etc. which are copied across. If everything that is important to you about the object has been successfully cloned, then it is a suitable model object.
+In Infusion, a tree of objects has the status of being a Model Object if it can successfully be copied by the framework utility function `fluid.copy()`. 
+This function duplicates an object tree in a very clear way: each object is cloned recursively, except for immutable values such as strings, numbers, functions, etc. which are copied across. 
+If everything that is important to you about the object has been successfully cloned, then it is a suitable model object.
 
-Note that this does not prohibit functions, DOM nodes, etc. from appearing in a model, just so long as the meaning of the object tree is not destroyed through the operation of `fluid.copy()`. The point here is that Infusion doesn't impose a formal contract on model objects: as long as `fluid.copy()` succeeds, you're good. For reference, the current implementation of `fluid.copy()` simply forwards the operation to [`jQuery.extend(true, ...)`](http://docs.jquery.com/Utilities/jQuery.extend). In practice, your model objects will often be JSON-compatible data suitable for sharing between your JavaScript code and other services on the Web.
+We recommend that all objects stored in the `model` area of a component conform to an even stricter definition - that they are equivalent to their form as serialised as JSON. This implies that they
+consist of only the following types: `Array`, `Object`, `String`, `Number` and `Boolean`. In addition, the following values for `Number` should not be used - `Infinity`, `-Infinity` and `NaN`. The value
+`undefined` also can not be safely stored within a model.
 
 An analogy from another environment, Java, is a [POJO](http://en.wikipedia.org/wiki/POJO) or "Plain Old Java Object". The J could just as easily be substituted for "JavaScript" in this case.
```

Comment:
Is this analogy still useful? I imagine Java programmers still think of POJOs as more "object-oriented" than what an Infusion model is (e.g. POJOs still may well have getters and setters)